M2/Macaulay2/packages/Tableaux.m2

```diff
@@ -0,0 +1,69 @@
+newPackage(
+    "Tableaux",
+    Version => "1.0",
+    Date => "July 22, 2025",
+    Authors => {
+	{Name => "John Graf", Email => "jrgraf@alumni.ncsu.edu", HomePage => "https://j-graf.github.io/"}},
+    Headline => "a package for constructing skew tableaux",
+    Keywords => {"Combinatorics"},
+    AuxiliaryFiles => true,
+    DebuggingMode => false,
+    PackageImports => {"Permutations"}--,
+    --Configuration => {"Convention" => "English", "TexPackage" => "aTableau"}
+    )
+
+export {"SkewTableau", "skewTableau",
+        "youngDiagram", "ferrersDiagram", "drawInnerShape",
+        "skewShape", "standardize", "rowEntries", "columnEntries", "rowRange", "columnRange",
+        "isSkew", "isWeaklyDecreasing", "isNonnegative",
+        "toPosition", "toIndex", "positionList", "applyEntries", "applyPositions",
+        "verticalConcatenate", "shift", "unshift",
+        "boxContent", "hookLength",
+        }
+
+export {"YngTableau", "yngTableau",
```

Comment:
Is there a particular reason you decided against `YoungTableau` and `youngTableau`? The [Package Writing Style Guide](https://github.com/Macaulay2/M2/wiki/Package-Writing-Style-Guide) recommends to "Avoid acronyms in identifier names."